src/utils/rest-api.js

```diff
@@ -483,6 +483,24 @@ export function getNameCandidate(directoryUuid, elementName, type) {
     });
 }
 
+/**
```

Comment:
you should specify everywhere below we are talking about a case, because this function is case-related.
ex: getCaseOriginalName(caseUuuid) ... etc

Reply:
done